docs/advanced-topics/best-practices/best-practices-dockstore.rst

```diff
@@ -0,0 +1,107 @@
+
+.. image:: /assets/images/docs/manage-labels.png
+
+Once you click the text, you'll be able to add or remove labels for your entry. To get ideas for what to use as labels, look at what others are using on the search page.
+Also note that each label cannot have spaces and must be all lowercase. If you want a label to be two words, use a hyphen.
+
+Another option for making your work more visible is using our :doc:`Organization and Collection <../organizations-and-collections>` pages.
+An Organization is a landing page for your collaborations, institutions, consortiums, companies, etc. Here you can explain the work and goals your group has,
+and highlight your tools and workflows by adding them to a Collection.
+
+Making It Easier to Understand
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Once a user has found your entry, they need to understand what it does and how to use it! The most important thing you can do so others understand your work is to provide a thorough description.
+You can do this by filling out the metadata field as explained in the best practices tutorials linked above, or you can write one in the README file.
+Your description can be formatted using markdown. Once registered, it will be parsed by Dockstore and made available on the Info tab of an entry.
+Because it will be on the of first things a user will see when looking at your entry, you should make it as detailed as possible. Here is a list of items to write about:
+
+* About section.
+
+   * What does your tool or workflow do?
+   * Are you part of a bigger organization? What are some of their goals
+
+* How to Use Section.
+
+   * What are the system requirements? Minimum and recommended
+   * Describe the input and output files
+
+      * What are their names?
+      * What data do they contain?
+      * What is the format?
+
+   * Can you provide time and/or cloud cost estimations for running your tool/workflow with a given input?
+   * If available, link to tutorials using your entry.
+   * If available, link to a sample or complete dataset to use.
+
+* Related To Section
+
+  * Does your tool/workflow work together with other entries? If so, describe how they can be used together and provide links.
+  * Link out to other similar entries you think could be useful to others.
+
+* Contact Section
+* Citations
+
+   * Does your workflow employ packages that should be cited?
+
+Making It Easier to Use
+^^^^^^^^^^^^^^^^^^^^^^^
+Although it's not always possible, you should provide the input data needed to run the entry. You can do this a few different ways:
+
+* Provide links to the data needed in your description.
+* Have your entry download the input files using a link. You can do this directly in the descriptor files or by putting them in a test parameter file.
+* Have the files within the Docker image being used. If you do this, make sure you provide a description of the structure and expected files in the description above.
```

Comment:
I don't know that we should be recommending this... Did this use to be there in some form?

1. Depending on data, it can make the image size grow a lot
2. Tends to make it hard-coded for a particular set of data, although it could be overridden.

Reply:
No, it hasn't been mentioned before. I had asked @wshands for some suggestions and he mentioned this as an option.

Reply:
Including reference genome files, etc. in the Docker image is what is being done for the HuBMAP ATAC-seq pipeline; it dramatically increases the size of the Docker image but reduces the number of inputs needed. This was done to make it very easy for the team that runs the workflows to work with the pipeline. I don't know how common it is to use this technique.